tests/conftest.py

```diff
@@ -1,59 +1,58 @@
 from __future__ import annotations
 
-import asyncio
 import sys
-import threading
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Annotated, TypeVar
 
 import pytest
-import requests
+import pytest_asyncio
```

Comment:
Tests are currently broken. I was experimenting with `pytest_asyncio` to have async fixtures  that can be used to spin up an engine for the whole pytest session, but this is proving to be quite tricky.